WcaOnRails/db/migrate/20190728084145_add_wrc_fields_to_delegate_reports.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddWrcFieldsToDelegateReports < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Since you added a migration, there should be a corresponding change to our `structure.sql` file. Is it possible you forgot to `git add` that change?

Reply:
Yep! Just pushed that change. Good catch!